src/styles/card.css

```diff
@@ -0,0 +1,100 @@
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  background: var(--white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  height: 134px;
+  width: 160px;
+  align-self: center;
+  margin-bottom: 40px;
+}
+
+.card__tittle {
+  font-size: var(--m-font-size);
+  font-weight: var(--m-font-weight);
+  line-height: 18px;
+  color: var(--main-accent-color);
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  line-height: 14px;
+  margin-bottom: 16px;
+  font-size: var(--s-font-size);
+  color: var(--secondary-color);
+}
+
+.card__score {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-size: var(--s-font-size);
+  line-height: 14px;
+  color: var(--main-accent-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: var(--m-font-size);
+  line-height: 18px;
+  color: var(--secondary-color);
+}
+
+.card__price-cost {
+  font-size: var(--xl-font-size);
+  font-weight: var(--l-font-weight);
+  line-height: 14px;
+  color: var(--main-accent-color);
+}
+
+.card__button-buy {
+  width: 166px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid transparent;
+
+  font-size: var(--l-font-size);
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: var(--white-color);
+  font-weight: var(--l-font-weight);
+  padding: 11px 0;
+
+  transition: 0.3s;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated

Reply:
Check @Rina07 comment, there are right way how to use transition